src/plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -33,6 +33,9 @@ export interface Series {
   split: string;
   stack?: boolean;
   type: string;
+  bars?: Record<any, any>;
+  lines?: Record<any, any>;
+  points?: Record<any, any>;
```

Comment:
Can we strengthen all these `any` types? looks like we perfectly know what they represent